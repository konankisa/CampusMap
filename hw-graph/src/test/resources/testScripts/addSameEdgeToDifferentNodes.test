# Creates an empty graph, adds 3 nodes to it,
# and adds the same edge to different nodes,
# then checks the children

# Create a graph
CreateGraph graph

# Add a node to graph
AddNode graph node1

# Add another node to graph
AddNode graph node2

# Add a third node to graph
AddNode graph node3

# Add an edge from node1 to node2 in graph
AddEdge graph node1 node2 edge1

# Add the same edge from node1 to node3 in graph
AddEdge graph node1 node3 edge1

# List the children of the parent node
ListChildren graph node1