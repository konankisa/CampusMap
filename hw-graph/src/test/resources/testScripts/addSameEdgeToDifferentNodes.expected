# Creates an empty graph, adds 3 nodes to it,
# and adds the same edge to different nodes,
# then checks the children

# Create a graph
created graph graph

# Add a node to graph
added node node1 to graph

# Add another node to graph
added node node2 to graph

# Add a third node to graph
added node node3 to graph

# Add an edge from node1 to node2 in graph
added edge edge1 from node1 to node2 in graph

# Add the same edge from node1 to node3 in graph
added edge edge1 from node1 to node3 in graph

# List the children of the parent node
the children of node1 in graph are: node2(edge1) node3(edge1)